### PR TITLE
Incremental changes for istiod

### PR DESCRIPTION
**Pull request by costinm**
- remove the SDS startup in pilot - the feedback is to start with SDS in
agent, pilot will only sign certs.
- cleanup a bit the startup
- use port 8080 for http - istioctl expects it.
- add the original Dockerfile - while it is getting integrated into the
istio build system ( can be useful after as well, it's very easy)

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

File: galley/pkg/crd/validation/validation.go [hunk 1]
```diff
@@ -72,17 +73,20 @@ func webhookHTTPSHandlerReady(client httpClient, vc *WebhookParameters) error {
 }
 
 //RunValidation start running Galley validation mode
-func RunValidation(ready chan<- struct{}, stopCh chan struct{}, vc *WebhookParameters, kubeConfig string,
-	livenessProbeController, readinessProbeController probe.Controller) {
+func RunValidation(ready chan<- struct{}, stopCh chan struct{}, vc *WebhookParameters, kubeInterface *kubernetes.Clientset, kubeConfig string, livenessProbeController, readinessProbeController probe.Controller) {
```
Comment (ayj) [inline galley/pkg/crd/validation/validation.go]: Can you replace kubeconfig with kubeInterface? We don't need to support both.

**Review by ayj — APPROVED** [review]

**Comment by sbezverk** [comment]
@costinm I am a bit confused, I was under impression that no webhook related work has already being done, also I thought the direction was to have as a separate package and call its functions from istiod while have webhook in Galley disabled.
Could you please keep work list and already taken tasks updated, so no stepping on other people toes would happen.

**Comment by howardjohn** [comment]
@costinm you have some merge conflicts

**Pull request merged**